src/Motors/MotorModule.c

```diff
@@ -0,0 +1,262 @@
+#include "MotorModule.h"
+#include "/time/timeInterrupt.h"
+#include <avr/io.h>
+#include <avr/sfr_defs.h>
+#include <avr/delay.h>
+#include <stdbool.h>
+
+/* ========================= Setup ================================= */
+#define blinkInterval 400
+uint32_t currentIndicater = 0;
+uint32_t previousIndicater = 0;
+
+// debug will allow the run of tests
+void initMotorModule(bool test) {
+	// Control Pins
+	DDRB |= 1<<PINB0 | 1<<PINB3 | 1<<PINB1 | 1<<PINB2;	// IN1/2 | IN3/4 | ENA | ENB
```

Comment:
The order is listed as the comment describes. First the direction pins then the pwm Pins. The order doesn't have to be MSB to LSB because after each or you describe what bit to shift.